src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -153,6 +156,27 @@ public void onSpoutOpen(
 
         createSidelineTriggers();
 
+        final long refreshIntervalSeconds;
+
+        // This should be set by SidelineConfig.setDefaults(), but just in case we want to avoid the NullPointer
+        if (spoutConfig.containsKey(SidelineConfig.REFRESH_INTERVAL_SECONDS)) {
+            refreshIntervalSeconds = ((Number) spoutConfig.get(SidelineConfig.REFRESH_INTERVAL_SECONDS)).longValue();
+        } else {
+            refreshIntervalSeconds = 600;
+        }
+
+        final long refreshIntervalMillis = TimeUnit.SECONDS.toMillis(refreshIntervalSeconds);
+
+        // Repeat our sidelines check periodically
+        final Timer timer = new Timer();
+        timer.scheduleAtFixedRate(new TimerTask() {
+            @Override
+            public void run() {
+                loadSidelines();
```

Comment:
You may want to put a try/catch or osmething around this.  If loadSidelines() ever throws an exception, it will never fire again after that.